pkg/manager/member/pump_member_manager.go

```diff
@@ -346,6 +375,16 @@ func getNewPumpStatefulSet(tc *v1alpha1.TidbCluster, cm *corev1.ConfigMap) (*app
 		},
 	}
 
+	if tc.IsTLSClusterEnabled() {
+		volumes = append(volumes, corev1.Volume{
+			Name: pumpCertVolumeMount, VolumeSource: corev1.VolumeSource{
+				Secret: &corev1.SecretVolumeSource{
+					SecretName: controller.PumpMemberName(tc.Name),
```

Comment:
The `SecretName` you get here is not the secret name you created in the shell script, right?
Did you test the case the pump is created with CR?

Reply:
no this is manager by certcontrol